### PR TITLE
Pherpat

### PR DESCRIPTION
**Pull request by pherpat**

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -81,20 +83,27 @@ function searchIMDB(q) {
...
             }
             else {
                 $("#movie-title").text(data.Title);
                 $("#movie-plot").text(data.Plot);
                 $("#watch-trailer").addClass("visible");
                 $("#add-movie").addClass("visible");
-                searchYouTube(q + " " + data.Year);
+                $("#cardHolder").show();
+                $(".cardBtnHolder").show(); // show add button
+                // searchYouTube(q + " " + data.Year); // comment it out when youtube quote is full
```
Comment (bryanjeremysmith) [inline assets/js/script.js]: I wonder if we should keep it uncommented out when committing to GitHub, and then make the change if it ends up affecting you?

**Review by bryanjeremysmith — APPROVED** [review]
Thank you!

File: assets/js/script.js (hunk 1 above)
Comment (pherpat) [reply assets/js/script.js]: oops sorry I forgot to put it back. Had to disable youtube so I could do testing.

Comment (bryanjeremysmith) [reply assets/js/script.js]: Sure thing I made the change during the merge!

Comment (pherpat) [reply assets/js/script.js]: yay!! Thank you!

**Pull request merged**